server/src/main/scala/actors/ShiftsActorBase.scala

```diff
@@ -24,20 +25,16 @@ case object ClearState
 
 case object GetFeedStatuses
 
-case class PointInTimeQuery(pointInTime: MillisSinceEpoch, query: DateRangeLike)
+case class PointInTimeQuery(pointInTime: MillisSinceEpoch, query: DateRangeLike) extends PortStateRequest
 
 trait DateRangeLike {
   val from: MillisSinceEpoch
   val to: MillisSinceEpoch
 }
 
-case class GetPortState(from: MillisSinceEpoch, to: MillisSinceEpoch) extends DateRangeLike
+case class GetFlightsForTerminal(from: MillisSinceEpoch, to: MillisSinceEpoch, terminal: Terminal) extends DateRangeLike with FlightsRequest
```

Comment:
check if we need the FlightsRequest trait

Reply:
Removed